caliban/docker.py

```diff
@@ -545,6 +561,19 @@ def _dockerfile_template(
                                      gid,
                                      adc_path=adc_path,
                                      credentials_path=credentials_path)
+  if inject_notebook.value != 'none':
+    install_lab = inject_notebook == NotebookInstall.lab
+    if dlvm is None:
+        dockerfile += _notebook_entries(lab=install_lab, version=jupyter_version, dlvm=False)
+    else:
+        dockerfile += _notebook_entries(lab=install_lab, version=jupyter_version, dlvm=True)
+
+  dockerfile += """
+
+USER {uid}:{gid}
```

Comment:
nice! Now that we're on python 3.6, we can actually make this

```python
dockerfile += f"""
USER {uid}:{gid}
"""
```

using [f-strings](https://realpython.com/python-f-strings/).